scripts/keeper.ts

```diff
@@ -45,9 +48,13 @@ export async function flush(config: Config) {
     Date.now() > lastFlushTimestamp + config.flushDelay
   ) {
     console.log("Sending `flush` transaction");
-    const { transaction_hash } = await l2WormholeGateway.flush(
+    const { amount } = await l2TeleportGateway.estimate(
```

Comment:
There is an estimateFee version of this method generated by TypeChain:
l2TeleportGateway.estimateFee.flush